js/components/Chart.vue

```diff
@@ -1,23 +1,32 @@
 <template>
   <div :id="chartId" class="vizr-chart-container">
     <div class="vizr-chart-header">
-      <h3 data-description="title">{{ chartDef.title }}
-      </h3>
-      <p v-if="hasDescription" data-description="description"><em>{{ chartDef.description }}</em></p>
+      <h3 data-description="title">{{ chartDef.title }}</h3>
+      <p v-if="hasDescription" data-description="description">
+        <em>{{ chartDef.description }}</em>
+      </p>
       <p class="chart-controls">
-        <a href="#" data-description="reload" role="button" @click.prevent="reloadChart">{{ messages.actions.reload }}
+        <a href="#" data-description="reload" role="button" @click.prevent="reloadChart">
+          {{ messages.actions.reload }}
           <i class="fas fa-sync-alt" :title="messages.actions.reload"></i>
         </a>
-        <a :href="formIdSelector" role="button" data-toggle="collapse" data-description="edit" v-if="canEdit">{{ messages.actions.edit }}
-          <i class="far fa-edit"></i>
+        <a
+          :href="formIdSelector"
+          role="button"
+          data-toggle="collapse"
+          data-description="edit"
+          v-if="canEdit"
+          >
```

Comment:
The dangling angle bracket is a little odd.

Reply:
Agreed. I can see if the JSX brackets setting alters this behavior.

Reply:
And the answer is no, it doesn't.

Reply:
That's a bummer, it does look out of place.